cluster-autoscaler/cloudprovider/clusterapi/clusterapi_unstructured_test.go

```diff
@@ -396,12 +404,24 @@ func TestAnnotations(t *testing.T) {
 			t.Errorf("expected %v, got %v", cpuQuantity, cpu)
 		}
 
+		if deprecatedCpu, err := sr.InstanceCPUCapacityAnnotation(); err != nil {
+			t.Fatal(err)
+		} else if deprecatedCpuQuantity.Cmp(deprecatedCpu) != 0 {
+			t.Errorf("expected %v, got %v", deprecatedCpuQuantity, deprecatedCpu)
+		}
+
 		if mem, err := sr.InstanceMemoryCapacityAnnotation(); err != nil {
 			t.Fatal(err)
-		} else if memQuantityAsBytes.Cmp(mem) != 0 {
+		} else if memQuantity.Cmp(mem) != 0 {
 			t.Errorf("expected %v, got %v", memQuantity, mem)
 		}
 
+		/* if deprecatedMem, err := sr.InstanceMemoryCapacityAnnotation(); err != nil {
+			t.Fatal(err)
+		} else if deprecatedMemQuantityAsBytes.Cmp(deprecatedMem) != 0 {
+			t.Errorf("expected %v, got %v", deprecatedMemQuantity, deprecatedMem)
+		} */
```

Comment:
@elmiko I am talking about here! it's still failing with `expected {i:{value:1024 scale:0} d:{Dec:<nil>} s:1024 Format:DecimalSI}, got {i:{value:1024 scale:0} d:{Dec:<nil>} s:1024 Format:DecimalSI}`. Going to step away from it briefly and take a look again shortly